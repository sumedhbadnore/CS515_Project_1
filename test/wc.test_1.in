This is a sample text file.

It has multiple lines.


A



few lines contains words and few are blank.