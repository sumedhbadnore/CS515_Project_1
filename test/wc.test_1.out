 4 5 20 test/wc.test_1.in